jflex/src/main/java/jflex/Emitter.java

```diff
@@ -1221,6 +1224,7 @@ private void emitEOFVal() {
         print("                System.out.println(");
         if (scanner.lineCount) print("\"line: \"+(yyline+1)+\" \"+");
         if (scanner.columnCount) print("\"col: \"+(yycolumn+1)+\" \"+");
+        if (scanner.charCount) print("\"char:\" + yychar + ");
```

Comment:
white space missing

Reply:
In this case, the previous output has a whitespace after, so this one should be fine.

Reply:
but there is one missing afterwards. Ok, will add.